imageCaptioningExperiments/clientHttpForImageCaptioning_with_text.py

```
@@ -0,0 +1,174 @@
+            "data": [json.dumps(input_payload)],
+        }
+    ],
+    "outputs": [
+        {
+            "name": "output"
+        }
+    ],
+}
+
+# Call the API
+response = requests.post(
```

Ruff (0.14.2)

Probable use of `requests` call without timeout

(S113)